src/Snppts/Snippets/Covid19Live.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Snppts.Authors;
+using Snppts.Extensions;
+using Snppts.Infrastructure;
+using System;
+using System.Collections.Generic;
+
+namespace Snppts.Snippets
+{
+    public class Covid19Live : IAmASnippet
+    {
+        public string Slug => "covid19-live";
+        public string Title => "Covid-19 Live";
+        public string Description => "Developed in XamarinForms with the aim of tracking Covid-19 cases with data from affected countries, including Brazil.";
+        public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("https://github.com/Altevir/xamarinforms-covid19");
```

Comment:
No need for the github domain. 
Just `Altevir/xamarinforms-covid19` is enough

Reply:
ops, sorry, i will fix.

Reply:
removed and committed =)